Expensify/App
# Typing indicator ellipses

### Details

- Added ellipses for long emails in typing indicator

### Fixed Issues

$ https://github.com/Expensify/App/issues/5623

### Tests

1. Tested email long email and display name
2. Tested for small email
3. Tested with multiple users


### QA Steps

1. Go to chat with any account A (or with a long email)
2. Go to chat with account B
3. Send chats from Account A to Account B, the typing indicator should show the trimmed username along with the `is typing` word visible
4. Test the same with small email, it shouldn't trim the email
5. For two or more users it should show `Multiple users typing...`

### Tested On


### Screenshots


#### Web

<img width="1148" alt="web-long-email-without-ellipses" src="https://user-images.githubusercontent.com/57435789/137788592-46c58827-a7c2-40a3-8229-c9971c15b0e7.png">
<img width="760" alt="web-long-email-with-ellipses" src="https://user-images.githubusercontent.com/57435789/137788601-6e764d6f-2ff5-4b78-b3a5-5c70c1432a6b.png">
<img width="690" alt="web-normal-email" src="https://user-images.githubusercontent.com/57435789/137788603-d92ce161-d421-4c35-a230-cd90e31ffaba.png">


#### Mobile Web

<img width="331" alt="mweb-email-ellipses" src="https://user-images.githubusercontent.com/57435789/137789193-efde4602-3d26-4eca-a848-d30e66da4611.png">

#### Desktop

<img width="881" alt="desktop-email" src="https://user-images.githubusercontent.com/57435789/137789237-3c79a8bc-f36f-475c-97aa-6df1fa56fc90.png">


#### iOS

<img width="342" alt="ios-email-ellipses" src="https://user-images.githubusercontent.com/57435789/137789775-a34f721a-e6f8-4031-bef7-f9ac9080d14b.png">

#### Android

<img width="318" alt="android-email-ellipses" src="https://user-images.githubusercontent.com/57435789/137789284-c7ab8415-d237-4cc4-bf13-2baaf85355a2.png">
